src/client/java/teammates/client/scripts/DataMigrationForSessions.java

```diff
@@ -26,70 +24,42 @@
  * </ul>
  * </p>
  */
-public class DataMigrationForSessions extends DataMigrationBaseScript<Key<FeedbackSession>> {
+public class DataMigrationForSessions extends DataMigrationWithCheckpointForEntities<FeedbackSession> {
 
     public static void main(String[] args) throws IOException {
         new DataMigrationForSessions().doOperationRemotely();
     }
 
-    /**
-     * {@inheritDoc}
-     */
     @Override
-    protected boolean isPreview() {
-        return true;
+    protected Query<FeedbackSession> getFilterQuery() {
+        return ofy().load().type(FeedbackSession.class);
     }
 
-    /**
-     * {@inheritDoc}
-     */
     @Override
-    protected List<Key<FeedbackSession>> getEntities() {
-        return ofy().load().type(FeedbackSession.class).keys().list();
+    protected boolean isPreview() {
+        return true;
     }
 
-    /**
-     * {@inheritDoc}
-     */
     @Override
-    protected boolean isMigrationNeeded(Key<FeedbackSession> sessionKey) {
-        FeedbackSession session = ofy().load().key(sessionKey).now();
-        try {
-            Field followingCourseTimeZoneField = session.getClass().getDeclaredField("wasFollowingCourseTimeZone");
-            followingCourseTimeZoneField.setAccessible(true);
-            return !followingCourseTimeZoneField.getBoolean(session);
-        } catch (ReflectiveOperationException e) {
-            return true;
-        }
```

Comment:
It is not run. I think the logic is useless as all `wasFollowingCourseTimeZone` was `false` currently.

https://github.com/TEAMMATES/teammates/blob/f5f345b8169e9a0f677ef4cbc8d1854b621fbd8e/src/main/java/teammates/storage/entity/FeedbackSession.java#L83-L85

Reply:
Following the PR #8759, `isFollowingCourseTimeZone` is true for all new sessions created after the PR. These sessions don't need to be migrated. But the PR is a bit convoluted and use another auxilary property `wasFollowingCourseTimeZone` to decide whether a session need to be migrated.

I think that this `wasFollowingCourseTimeZone` should be removed, and the migration script decide whether a session should be migrated based on `isFollowingCourseTimeZone`

Reply:
>  all `wasFollowingCourseTimeZone` was `false` currently.

Hmm this is a bit strange though. From my understanding of the logic of #8759, if a session is edited, i.e. loaded from the database, then it will have `wasFollowingCourseTimeZone` set to `true`. So if it is saved, this field should be modified to `true`. Could it be the case that nobody ever edit a feedback session in the past 4 months?